src/com/lilithsthrone/controller/xmlParsing/XMLLoadException.java

```diff
@@ -0,0 +1,18 @@
+package com.lilithsthrone.controller.xmlParsing;
+
+/**
+ * @author BlazingMagpie@gmail.com (or ping BlazingMagpie in Discord)
+ * @since 0.2.5.8
+ * 
+ * <p>Exception thrown when XML file fails to load, with details for exact cause.</p>
+ */
+public class XMLLoadException extends Exception{
+    /**
+     * <p>Contruct an exception from given cause, giving simplified explanation of what happened with possiblity to access details of root cause</p>
```

Comment:
this is a noise comment and can go

If you disagree, that's okay; fix the typos:
Contruct - Construct
"giving simplified explanation" -> "giving a simplified explanation"
possiblity -> the possibility